ddp-workspace/projects/ddp-pancan/src/app/components/header/header.component.scss

```diff
@@ -145,13 +158,57 @@
     }
 }
 
-@media screen and (max-width: 945px) {
+.project-logo {
+    display: flex;
+    margin-right: 15px;
+
+    &__text {
+        color: $text-color;
+        line-height: 18px;
+        padding-left: 7px;
+        letter-spacing: -0.5px;
+    }
+}
+
+.header-link {
+    color: $text-color;
+    font-family: $primary-font-medium;
+    letter-spacing: -0.015rem;
+    font-size: 1.5rem;
+    line-height: 3rem;
+}
+
+.join-button {
+    @extend .btn-common;
+    @extend .btn-header;
+    margin-left: 15px;
+}
+
+@media screen and (min-width: $desktop-breakpoint) {
     .hamburger {
+        display: none;
+    }
+
+    .panel-controls,
+    .panel-opened .panel-controls {
+        display: none;
+    }
+
+    .header-content .nav {
         display: flex;
     }
 
-    .nav,
+    .header {
+        z-index: 1;
```

Comment:
#minor
not necessary (already exists on line 9)

Reply:
removed